03.bowling/bowl.rb

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+score = ARGV[0]
+scores = score.split(',')
+shots = []
+scores.each do |s|
+  if s == 'X'
+    shots << 10
+    shots << 0
+  else
+    shots << s.to_i
+  end
+end
+
+frames = []
+shots.each_slice(2) do |s|
+  frames << if s[0] == 10
+              [s[0]]
+            else
+              s
+            end
+end
+
+point = 0
+
+(0..frames.size - 1).each do |i|
```

Comment:
```suggestion
frames.each_with_index do |current_frame, i|
```

each_with_indexを活用しましょう

加えて、sumメソッドを使うとブロックの中がよりシンプルに書けます。

```suggestion
point = frames.each_with_index.sum do |current_frame, i|
```